src/app/components/Document.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { AfterRoot, AfterData } from '@jaredpalmer/after';
+
+class Document extends React.Component {
+  static defaultProps = {
+    data: {},
+  }
+
+  static propTypes = {
+    helmet: PropTypes.objectOf(PropTypes.any).isRequired,
+    assets: PropTypes.objectOf(PropTypes.any).isRequired,
+    data: PropTypes.objectOf(PropTypes.any),
+  }
+
+  static async getInitialProps({ assets, data, renderPage }) {
+    const page = await renderPage();
+    return { assets, data, ...page };
+  }
+
+  render() {
+    const { helmet, assets, data } = this.props;
+    const htmlAttrs = helmet.htmlAttributes.toComponent();
+
+    return (
+      <html lang="en-GB" {...htmlAttrs}>
```

Comment:
Since we're already passing `lang` as a prop in https://github.com/bbc/simorgh/blob/latest/src/app/components/Article.jsx#L9, we should remove the hardcoded value from here.

Reply:
The linter complains if we don't have a lang attr on the <html> element. Therefore this is a default which gets overridden. Happy to add the linting ignore if wanted

Reply:
If we remove this hard-coded `lang` prop, we rely on always serving a `lang` in a component that is served. This hard-coded value acts as a default.

Reply:
In light of this, leaving it this way sounds good to me!